clients/filesystem-hadoop3/src/main/java/com/datastrato/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -51,6 +52,13 @@ public class GravitinoVirtualFileSystem extends FileSystem {
   private Cache<NameIdentifier, Pair<Fileset, FileSystem>> filesetCache;
   private ScheduledThreadPoolExecutor scheduler;
 
+  // The pattern is used to match gvfs path. The scheme prefix (gvfs://) is optional.
```

Comment:
The scheme prefix should be "gvfs://fileset", right, not just "gvfs://"?

Reply:
@jerryshao You are right. I have fixed it. Please take a look. Thanks.